SDWebImageSwiftUI/Classes/WebImage.swift

```diff
@@ -66,10 +66,12 @@ public class WebImagePlayer: ObservableObject {
     
     var selfId = UUID()
     
+    var isCacheImage = false
```

Comment:
Potentially re-name to `supportsFakeLoadingDelay`

Reply:
done